requirements.txt

```diff
@@ -3,6 +3,6 @@ botocore
 cryptography >= 0.8
 httplib2
 paramiko
-pghoard >= 2.1.0
 PyMySQL >= 0.9.2
 PySocks
+rohmu
```

Comment:
It won't work with any version of `rohmu`, would be better to specify `rohmu >= 1.0.2`